src/views/RealTimeEvents/Blocks/content-filter-block.vue

```diff
@@ -34,7 +34,7 @@
     filter.value.isUserUsingGraphqlQuery = false
   })
 
-  const filterMode = ref('Wizard')
+  const filterMode = ref('Advanced')
```

Comment:
This is for setting the default filter that will be displayed when opening the window?
Think the default should be more user friendly.